charts/zero-notifcation-service/templates/external-secret.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.externalSecret.enabled -}}
```

Comment:
Shouldn't these files be conditionally templated in the module.yaml rather than just excluding all the content in the file?